src/dbnode/client/types.go

```diff
@@ -148,20 +148,23 @@ type PeerBlocksIter interface {
 type AdminSession interface {
 	Session
 
-	// Origin returns the host that initiated the session
+	// Origin returns the host that initiated the session.
 	Origin() topology.Host
 
-	// Replicas returns the replication factor
+	// Replicas returns the replication factor.
 	Replicas() int
 
-	// TopologyMap returns the current topology map
+	// TopologyMap returns the current topology map. Note that the session
```

Comment:
i think you can remove this method now. Unused afaict

Reply:
one of our tests uses it

Reply:
fair enough, i think we should move the tests to use the t-channel health/bootstrapped endpoint. opened https://github.com/m3db/m3/issues/1016 to followup